packages/remediations/src/redux/actions/resolution-actions.js

```diff
@@ -0,0 +1,26 @@
+import * as api from '../../api';
+import { FETCH_RESOLUTIONS } from '../action-types';
+import transform from 'lodash/transform';
+
+export const fetchResolutions = (issues = []) => ({
+    type: FETCH_RESOLUTIONS,
+    payload: async () => {
```

Comment:
The payload shouldn't be a function, can you make this into a const above this action and create the action as
```JS
export const fetchResolutions = (issues = []) => ({
  type: FETCH_RESOLUTIONS,
  payload: batchResolution(issues)
});
```